docs/job-seekers-guide/categorize-contributions.md

```diff
@@ -0,0 +1,190 @@
+---
+id: job-seekers-guide-how-to-organize-your-open-source-contributions
+title: "How to organize your open source contributions for jobs"
```

Comment:
Also, not super in love with the title at the moment so I would be open to suggestions

Reply:
Maybe something like:

- Keeping Track Open Source Contributions for Job Seeking
- Organizing Open Source Contributions to Make Your Resume Stand Out
- Making Resume Stand Out through Open Source Contributions

Also @jdwilkin4, for consistency throughout the docs, let's capitalize all titles 😊

Reply:
A couple more options

- "How to Effectively Track and Showcase Your Open Source Contributions for Career Advancement"
- "How to Build a Compelling Portfolio with Your Open Source Work"
- "How to Translate Open Source Contributions into Career Opportunities"
- "How to Document Your Open Source Journey for Professional Success"
- "How to Leverage Your Open Source Experience in Your Resume"

Reply:
How about this one:

- “How to Leverage the Power of Open Source in your Job Search”
- “How to use Open Source in your Job Search”